content/200-concepts/100-components/01-prisma-schema/07-referential-actions.mdx

```diff
@@ -0,0 +1,315 @@
+
+### New behavior
+
+With the feature flag **enabled**, the default value of `ON DELETE` is [`Restrict`](#restrict) (`ON DELETE RESTRICT`).
+
+The Prisma Client will **not** perform any runtime checks before deleting related records with `delete()` or `deleteAll()`. This means the deletion will succeed or fail according to the underlying foreign key relations and any referential actions set. 
+
+By default the deletion will be prevented by the database because of the **new** default action, [`Restrict`](#restrict) (`ON DELETE RESTRICT`). To make it succeed you will need to manually add the [`Cascade`](#cascade) action. 
+
+#### Required relations
+
+| Clause      |  < 2.26.0       | 2.26.0 +   |
+| :---------- | :-------------- | :--------- |
+| `onDelete`  | `Cascade`       | `Restrict` |
+| `onUpdate`  | `Cascade`       | `Cascade`  |
+
+#### Optional relations
+
+| Clause      |  < 2.26.0       | 2.26.0 +   |
+| :---------- | :-------------- | :--------- |
+| `onDelete`  | `SetNull`       | `SetNull`  |
+| `onUpdate`  | `Cascade`       | `Cascade`  |
+
+<Admonition type="warning">
+
+Referential actions are **not currently** supported on [implicit many-many relations](./relations#implicit-vs-explicit-many-to-many-relations), instead you must create a [join table](./relations#how-to-use-a-relation-table-with-a-many-to-many-relationship) in your schema file and explicitly set the referential actions on the many-many relation.
+
+</Admonition>
+
+## Types of referential actions
+
+### Cascade
+
+When used with `onDelete`, The `Cascade` action deletes a record if a dependant record is deleted. 
+
+When used with `onUpdate`, it updates the relation scalar filed if the referenced scalar fields of the dependant record are updated.
+
+#### Example usage
+
+```prisma file=schema.prisma highlight=5;add|4;delete
+model Post {
+    id        Int     @id @default(autoincrement())
+    title     String
+    author    User    @relation(fields: [authorId], references: [id])
+    author    User    @relation(fields: [authorId], references: [id], onDelete: Cascade, onUpdate: Cascade)
+    authorId  Int
+}
+
+model User {
+  id    Int         @id @default(autoincrement())
+  posts Post[]
+}
+```
+
+<Admonition type="info">
+
+`onUpdate` defaults to `Cascade` when not set. This applies to both **optional** and **required** relations. `onDelete` defaults to `Cascade` on **required** relations.
+
+</Admonition>
+
+##### Result of using `Cascade`?
+
+If a `User` record is deleted then their posts are deleted too. If a users id is updated then the corresponding `authorId` is also updated
+
+<Admonition type="warning">
+
+The `Cascade` action prevents both update and deletion operations from completing if any records are connected
```

Comment:
What do you mean with this?

Reply:
Reading it back im not sure now. Will remove until/if i remember again